aten/src/ATen/native/cuda/TensorCompare.cu

```diff
@@ -129,6 +129,13 @@ void clamp_max_scalar_kernel_impl(TensorIterator& iter, Scalar max) {
   });
 }
 
+void isin_default_kernel_gpu(const Tensor& elements, const Tensor& test_elements, bool invert, const Tensor& out) {
```

Comment:
Add a comment for the CUDA version of the algorithm

Reply:
Will add a comment explaining the algo! Also, maybe I'm misunderstanding your statement, but this is just the generic composite approach.